src/test/java/Homework20.java

```diff
@@ -0,0 +1,40 @@
+import org.testng.annotations.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+    public class Homework20 extends BaseTest {
+        @Test(priority = 1, description = "Play a song and validate if song is playing")
+        public void addSongToPlaylist() throws InterruptedException {
+            String newSongAddNotificationMessage = "Added 1 song into";
+            provideEmail("catsaremyfavorite@gmail.com");
+            providePassword("te$t$tudent");
+            submit();
+            findSong("Take my Hand");
+            clickViewAllButton();
+            selectFirstSong();
+            clickAddToButton();
+            selectPlaylist();
+            Assert.assertTrue(getNotificationMessage().contains(newSongAddNotificationMessage));
+        }
+        @Test(priority = 2, description = "Play a song and validate if song is playing")
+        public void playSongTest() throws InterruptedException {
+            provideEmail("catsaremyfavorite@gmail.com");
+            providePassword("te$t$tudent");
+            submit();
+            usePlayerControls();
+            Assert.assertTrue(songIsPlaying());
+        }
+        @Test(priority = 3, description = "Delete an empty playlist")
+        public void deleteCreatedPlaylist() throws InterruptedException {
+            provideEmail("catsaremyfavorite@gmail.com");
+            providePassword("te$t$tudent");
+            submit();
+            selectEmptyPlaylist();
+            deletePlaylist();
+            getDeletePlaylistMessage();
+            Assert.assertTrue(getDeletePlaylistMessage());
+        }
+    }
+
```

Comment:
Please, **delete** `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.


![image](https://user-images.githubusercontent.com/104761482/222947458-8403f408-502a-442f-ac90-99465811130d.png)

Reply:
Okay, I'll clean it up - I realized I had to remove all of those from the BaseTest code; didn't know I had to remove it from the test code too!